src/styles/productCards.scss

```diff
@@ -0,0 +1,138 @@
+$sum-cards: 4;
+$card-width: 198px;
+$box-width: $card-width * $sum-cards +152;
+$height-conteiner: 406px * 2 + 50px;
+$color-dark: #060b35;
+$color-blue: #00acdc;
+
+.main {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  box-sizing: border-box;
+
+  &__content {
+    display: flex;
+    justify-content: flex-start;
+    flex-wrap: wrap;
+    max-width: $box-width;
+    flex: 0 0 auto;
+    max-height: $height-conteiner;
+  }
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-content: center;
+    width: $card-width;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    margin-left: 48px;
+    margin-bottom: 46px;
+
+    &:nth-child(-n+4) {
+      margin-top: 50px;
+    }
+
+    &:first-child {
+      margin-left: 0;
+    }
+    &:nth-child(5n) {
+      margin-left: 0;
+    }
+  }
+
+  &__image {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-bottom: 40px;
+  }
+
+  &_image-apple {
+    height: 134px;
+    width: 160px;
+    margin-top: 32px;
+  }
+
+  &__info {
+    margin-bottom: 4px;
+    padding: 0 16px;
+    line-height: 18px;
+    font-weight: 500;
```

Comment:
card should be a separate bem block

Reply:
agree, it should be in separate bem block